### PR TITLE
adding create namespace fxnality to reckoner to replace it in helm3

### PR DESCRIPTION
**Pull request by ejether**

### CODE REVIEW

File: reckoner/kube.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    Arguments:
+
+    namespace: The namespace to create
+
+    Returns True on success
+    Raises error in case of failure
+
+    """
+    print("HERE")
```
Comment (sudermanjr) [inline reckoner/kube.py]: I think this is a leftover debug statement.

File: reckoner/kube.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        )
+        return True
+    except Exception as e:
+        logging.error("Unable to create namespace in cluster! {}".format(e))
+        logging.debug(traceback.format_exc())
+        raise e
+
+
+def list_namespace_names():
+    """ Lists namespace in the configured kubernetes cluster.
```
Comment (sudermanjr) [inline reckoner/kube.py]: nit
```suggestion
    """ Lists namespaces in the configured kubernetes cluster.
```

File: reckoner/kube.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    No agruments
```
Comment (sudermanjr) [inline reckoner/kube.py]: sp.
```suggestion
    No arguments
```

**Comment by ejether** [comment]
Requested changes made.

**Pull request merged**